lms/templates/verify_student/webcam_photo.underscore

```diff
@@ -21,11 +21,11 @@
 
 <div class="controls photo-controls">
   <div class="control control-retake is-hidden" id="webcam_reset_button">
-    <a class="action action-redo"><%- gettext( "Retake Photo" ) %></a>
+    <a class="action action-redo" tabindex=1><%- gettext( "Retake Photo" ) %></a>
   </div>
   <div class="control control-do is-hidden" id="webcam_capture_button">
     <a class="action action-do">
-      <i class="icon fa fa-camera" aria-hidden="true"></i> <span class="sr"><%- gettext( "Take Photo" ) %></span>
+      <i class="icon fa fa-camera" aria-hidden="true" tabindex=1></i> <span class="sr"><%- gettext( "Take Photo" ) %></span>
```

Comment:
I think the tabindex should be set on the parent <a> tag instead of the icon shouldn't it?